lib/urn.rb

```diff
@@ -1,26 +1,88 @@
+def URN(urn)
+  URN.new(urn)
```

Comment:
One nice property of `URI()` is that you can use it as a type-coercion and it behaves as a no-op when you pass it an existing instance of `URI`.

What happens if we pass a `URN` to `URN()`?

```ruby
URN(URN('urn:foo:bar'))
```